lib/src/Uncertainty/Distribution/openturns/HistogramFactory.hxx

```diff
@@ -50,12 +50,18 @@ public:
                      const Scalar bandwidth) const;
   Distribution build(const Sample & sample,
                      const UnsignedInteger binNumber) const;
+  Distribution build(const Sample & sample,
+                     const Scalar first,
+                     const Point width) const;
   Distribution build() const;
   Histogram buildAsHistogram(const Sample & sample) const;
   Histogram buildAsHistogram(const Sample & sample,
                              const Scalar bandwidth) const;
   Histogram buildAsHistogram(const Sample & sample,
                              const UnsignedInteger binNumber) const;
+  Histogram buildAsHistogram(const Sample & sample,
+                             const Scalar first,
+                             const Point width) const;
```

Comment:
Here width should be passed by reference as it avoids costly copy, and it is implemented with a reference, hence the undefined symbol you get.

Reply:
You are right: OK.